projects/plugins/jetpack/modules/module-info.php

```diff
@@ -723,6 +723,25 @@ function jetpack_custom_jetpack_manage() {
 }
 add_action( 'jetpack_module_more_info_manage', 'jetpack_custom_jetpack_manage' );
 
+/**
+ * Post list info.
+ */
+function jetpack_post_list_link() {
+	echo esc_url( Redirect::get_url( 'jetpack-support-post-list' ) );
```

Comment:
I would recommend creating a redirect entry for this, pointing to the draft support doc for now.